package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "@demos-europe/demosplan-utils",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
Please don't increase the version in a PR. A version bump is not necessarily done via a single PR.

Reply:
f4eea7edb316ca0ac228a7cb8298cd551487d95d